modules/distr-tracing-config-otel-send-data-to-monitoring-stack.adoc

```diff
@@ -0,0 +1,39 @@
+////
+This module included in the following assemblies:
+-distr_tracing_install/distributed-tracing-deploying-otel.adoc
+////
+:_content-type: REFERENCE
+[id="distr-tracing-config-otel-collector_monitoring_{context}"]
+= Sending metrics to the monitoring stack
+
+The metrics from the {OTELName} can be consumed by the in-cluster monitoring stack. Refer to
+xref:../../monitoring/enabling-monitoring-for-user-defined-projects.adoc[enable monitoring for user defined projects] how to
```

Comment:
refs are not allowed in modules, only assemblies. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references

rather than including a new module here, it might make more sense to add a module in the monitoring section about "Consuming metrics from {OTELName}"?

Reply:
> rather than including a new module here, it might make more sense to add a module in the monitoring section about "Consuming metrics from {OTELName}"?

I think this content belongs more to the distributed tracing as this product uses some of the features of the monitoring stack (and there will be a bit more in the future).

Reply:
> refs are not allowed in modules, only assemblies

Thanks, I didn't know, I have moved the link to the upper-level assembly.